win32/src/win32evtlog.i

```diff
@@ -979,18 +979,6 @@ static PyObject *RenderEventValues(EVT_HANDLE render_context, EVT_HANDLE event)
 		if (err != ERROR_INSUFFICIENT_BUFFER) {
 			PyWin_SetAPIError("EvtRender", err);
 			goto cleanup;
-		} else {
-			// allocate buffer size
-			allocated_size = returned_size;
-			variants = (PEVT_VARIANT)malloc(allocated_size);
-			if (variants == NULL) {
-				PyErr_NoMemory();
-				goto cleanup;
-			}
-
-			Py_BEGIN_ALLOW_THREADS
-			bsuccess = EvtRender(render_context, event, EvtRenderEventValues, allocated_size, variants, &returned_size, &prop_count);
-			Py_END_ALLOW_THREADS
```

Comment:
By remove `else` block, I think it just means the literal `else` and not the contents of the block.

The `goto` statement on the line before the `else` means that having an else to control the flow is unnecessary.

The contents of the `else` block are essential to the correct behaviour of this function, otherwise memory won't be allocated and also the bsuccess field won't be set to the result of the `EvtRender` call.

Reply:
Fixed, thanks! How does it look now?

Reply:
That part looks fine.  I'll leave it to mark to comment on the rest of the PR.